README.md

```diff
@@ -60,6 +60,26 @@ We have also made a zero-shot evaluation of our models using data from 41 other
 | Belarus       |          88.77 |       93.00 | Kazakhstan    |          87.24 |       91.23 |
 | Ireland       |          86.35 |       87.49 |
 
+#### Noisy data
+The following table presents the accuracy on the 20 countries we used during training for both our models but for noisy 
+and incomplete data. We didn't test on the other 41 countries since we did not train and results were even lower.
```

Comment:
```suggestion
and incomplete data. We didn't test on the other 41 countries since we did not train on them and the results would most likely be even lower.
```